mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/automaticindexing/filter/PojoAutomaticIndexingTypeFilterContext.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.automaticindexing.filter;
+
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.util.common.annotation.Incubating;
+
+/**
+ * A context that helps with the indexing filter configuration.
+ * <p>
+ * Note, that only indexed types can be passed to this context's methods. Passing any other types will lead to an
```

Comment:
This is a curiously German syntax :) "Beachten Sie, dass [...]"
I don't think there's a comma here in English.

```suggestion
 * Note that only indexed types can be passed to this context's methods. Passing any other types will lead to an
```